task_manager_flutter/lib/controller/splash/splash_controller.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+
+import 'package:get/get.dart';
+import 'package:task_manager_flutter/data/model/user_model.dart';
+import 'package:task_manager_flutter/data/respository/auth_repository.dart';
+import 'package:task_manager_flutter/routes/app_page.dart';
+import 'package:task_manager_flutter/services/local/shared_prefs.dart';
+
+class SplashController extends GetxController {
+  final AuthRepository authRepository;
+
+  SplashController({required this.authRepository});
+  final isProgress = true.obs;
+
+  checkLoggedIn() async {
```

Comment:
pls remove this fund, we should not save user authentication info in local storage